src/ZooHomework/Main.java

```diff
@@ -1,29 +1,122 @@
+            animalsFileLoadArray.add("Elephant, Felipe, Pachyderm, Orejon, 14, 1800");
+            animalsFileLoadArray.add("Elephant, Felipe, Pachyderm, Orejon, 14, 1800");
+            animalsFileLoadArray.add("Lion, Nala, Feline, cat, 6, 65");
+            animalsFileLoadArray.add("Lion, Simba, Feline, brown mane, 5, 65");
+
+            for (String animal : animalsFileLoadArray) {
+                writer2.write(animal + "\n");
+            }
+            writer2.close();
+        } catch (IOException e) {
+            System.out.println("Could not write to file: " + e.getMessage());
+        }
+
+        ArrayList<ZooAreas> zooAreasFileReadArray = new ArrayList<>();
+        try {
+            BufferedReader reader1 = new BufferedReader(new FileReader("src/ZooHomework/zooAreas.csv"));
+            reader1.readLine();
+            String[] areas;
+            String line = reader1.readLine();
+            int i = 0;
+            while (line != null) {
+                areas = line.split(", ");
+                zooAreasFileReadArray.add(i, new ZooAreas(areas[0], Integer.parseInt(areas[1])));
+                i++;
+                line = reader1.readLine();
+            }
+            reader1.close();
+        } catch (IOException e) {
+            System.out.println("Could not access file: " + e.getMessage());
+        }
+
+        ArrayList<Animal> animalsFileReadArray = new ArrayList<>();
+        try {
+            BufferedReader reader2 = new BufferedReader(new FileReader("src/ZooHomework/animals.csv"));
+            reader2.readLine();
+            String[] animals;
+            String line = reader2.readLine();
+            int i = 0;
+
+            while (line != null) {
+                animals = line.split(", ");
+                if (animals[0].equals("Elephant")) {
+                    animalsFileReadArray.add(i, new Elephant(animals[1],
+                                                             animals[2],
+                                                             animals[3],
+                                                             Integer.parseInt(animals[4]),
+                                                             Integer.parseInt(animals[5])));
+                } else if (animals[0].equals("Lion")) {
+                    animalsFileReadArray.add(i, new Lion(animals[1],
+                                                         animals[2],
+                                                         animals[3],
+                                                         Integer.parseInt(animals[4]),
+                                                         Integer.parseInt(animals[5])));
+                }
+                i++;
+                line = reader2.readLine();
+            }
+            reader2.close();
+        } catch (IOException e) {
+            System.out.println("Could not access file: " + e.getMessage());
+        }
+
```

Comment:
Break point

Reply:
Agreed it needs a more complete solution. I could use some help fixing that.